routes/w3c/group.js

```diff
@@ -19,22 +19,37 @@ const API_KEY = env("W3C_API_KEY");
 /** @type {MemCache<Group>} */
 const cache = new MemCache(ms("2 weeks"));
 
+// Support non W3C shortnames for backward compatibility.
+const LEGACY_SHORTNAMES = new Map([
+  ["wai-apa", "apa"],
+  ["i18n", "i18n-core"], // more than 10 instances
+]);
+
```

Comment:
wai-apa -> "apa": 83907,
https://github.com/w3c/apa

i18n -> "i18n-core": 32113,
https://github.com/w3c/alreq, https://github.com/w3c/bp-i18n-specdev, https://github.com/w3c/charmod-norm, https://github.com/w3c/eurlreq, https://github.com/w3c/hlreq, https://github.com/w3c/i18n-activity, https://github.com/w3c/klreq, https://github.com/w3c/ltli, https://github.com/w3c/mlreq, https://github.com/w3c/sealreq, https://github.com/w3c/simple-ruby, https://github.com/w3c/tlreq, and many more